src/components/Api.js

```diff
@@ -0,0 +1,134 @@
+export class Api {
+  constructor({ baseUrl, headers }) {
+    // constructor body
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
```

Comment:
According to the task you need to pass a full `baseUrl` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `this._baseUrl` (not to duplicate the url)

Reply:
Every method now take "this.baseUrl"